src/visualization/Turi Create Visualization/src/user_interface/package.json

```diff
@@ -12,6 +12,8 @@
         "react": "^16.2.0",
         "react-dom": "^16.2.0",
         "react-json-pretty": "^1.7.5",
+        "react-vega": "^4.0.2",
+        "vega-lib": "^4.4.0",
```

Comment:
Make sure the vega-lib version matches the vega version. Come to think of it, I'm pretty sure vega is a superset of vega-lib -- do we need to add this dependency?